src/safeds/data/tabular/containers/_table.py

```diff
@@ -1708,6 +1708,22 @@ def inverse_transform_table(self, fitted_transformer: InvertibleTableTransformer
         """
         return fitted_transformer.inverse_transform(self)
 
+    def join(
+        self,
+        right_table: Table,
+        left_names: str | list[str],
+        right_names: str | list[str],
+        *,
+        mode: Literal["inner", "left", "outer"] = "inner",
+    ) -> Table:
```

Comment:
Docstring is missing, see [guidelines](https://library.safeds.com/en/stable/development/guidelines/documentation/).